libs/file-storage/src/lib/file-storage.service.ts

```diff
@@ -24,8 +24,15 @@ export class FileStorageService {
       throw new Error('Upload bucket not configured.')
     }
 
+    // To make sure we dont format the file extension when its present. If its not present return the filename
+    const splitFileName = filename.split('.')
```

Comment:
If `filename` is null or undefined then this will throw error. Do you want to test for that case before using `split()`?

Reply:
It doesnt hurt to check for it at least. Even though im not sure if you can upload an file with no filename. 
Belti og axlabönd!

Reply:
Throwing an error when filename is missing, we could also define the filename as 'unknown-file' or something to make sure that the upload doesnt break.